lib/standards/aria-roles.js

```diff
@@ -816,7 +819,7 @@ const ariaRoles = {
     superclassRole: ['section']
   },
   timer: {
-    type: 'widget',
+    type: 'structure',
```

Comment:
When you're adding roles to `valid-scrollable-semantics`, I think you can leave this one out. `role=timer` doesn't seem like it should be focusable.